src/services/filters.ts

```diff
@@ -0,0 +1,35 @@
+import { SelectableValue } from '@grafana/data';
+import { DetectedLabel } from './fields';
+import { ALL_VARIABLE_VALUE, LEVEL_VARIABLE_VALUE } from './variables';
+
+export enum FilterOp {
+  Equal = '=',
+  NotEqual = '!=',
+}
+
+// We want to show labels with cardinality 1 at the end of the list because they are less useful
+// And then we want to sort by cardinality - from lowest to highest
+export function sortLabelsByCardinality(a: DetectedLabel, b: DetectedLabel) {
```

Comment:
I'm still seeing everything sorted alphabetically, what am I missing?

Reply:
Labels? Because I haven't changed sorting in detected fields. And in Labels, I am just moving single cardinality labels to the bottom.

Reply:
Yeah, I plan to have a look at it after this. To see if I can easily fix it.